### PR TITLE
Sets dashboard counts invisible if empty (fixes #539)

### LINKED ISSUE #539: Home: Count of Home menu
I think it would be better make the count invisible if there is no added items or if count is 0.

device-2019-02-01-224853

### PR DESCRIPTION
**Pull request by jazdao**
Fixes #539 

dashboard

### CODE REVIEW

**Comment by rrijal53** [comment]
@inDepthh  fix code climate issue.

**Comment by jazdao** [comment]
Hi @rrijal53, I'm not quite sure what the best way is to refactor the conditional statements since each one refers to a different resource ID.  Do you have any tips?

**Comment by rrijal53** [comment]
@jazdao  you can create a function which takes , view and count as a parameter and put the condition inside that function. Something Like
``` 
public void hideCountIfZero(View vi, int count){
vi.setVisibility(count ==0 ? View.GONE : View.VISIBLE)
}
```
and call 
```
hideCountIfZero(v.findViewById(R.id.your_id), count);
```

**Comment by jazdao** [comment]
@rrijal53 Okay, thanks!

**Comment by rrijal53** [comment]
@jazdao still 3 issues exists. Fix them too.

File: app/src/main/java/org/ole/planet/myplanet/ui/dashboard/DashboardFragment.java [hunk 1]
```diff
@@ -208,28 +208,20 @@ public void onDestroy() {
 
     public void setCountText(int countText, Class c, View v) {
         if (c == RealmMyCourse.class) {
-            if (countText == 0) {
-                v.findViewById(R.id.count_course).setVisibility(View.INVISIBLE);
-            }
-            else {
-                ((TextView) v.findViewById(R.id.count_course)).setText(countText + "");
-            }
+            ((TextView) v.findViewById(R.id.count_course)).setText(countText + "");
```
Comment (yuviii) [inline app/src/main/java/org/ole/planet/myplanet/ui/dashboard/DashboardFragment.java]: Initialized the same view twice instead create the object of TextView and use it like
`TextView tv_count_course = v.findViewById(R.id.count_course);
hideCountIfZero(tv_count_course, countText);`

Comment (leomaxi) [reply app/src/main/java/org/ole/planet/myplanet/ui/dashboard/DashboardFragment.java]: Good feedback

**Pull request merged**